[New site] Content section (#7890)
elastic/eui: packages/website/docs/content/adapt-tone.mdx

---
id: content_tone
slug: content-tone
sidebar_position: 20
---

> # Tones

Reviewer: I know there's more than one tone, but it feels a little off to me to say "Tones" over "Tone." Maybe this is just a me problem 😄

Author: ```suggestion
# Tone
```

    - Take time to help the user understand how this feature benefits them.
    - Use visuals to support your text.

**Cases**

    Although it can be very visible because of the way it's written and the UI elements it shows in, the **stimulating** tone is used **quite rarely** overall. 
    
    We use it when we want to show new things to users, especially in:

    - Product tours
    - New feature announcements

## Informational

**Attributes**

    E=MC². Be:

    - Direct
    - Straight to the point
    - Neutral

    When using this tone:
    - Use a minimum of words.
    - Write for scanning and help users quickly locate what matters to them.
    - Use action verbs to: 
        - Label selection controls.
        - Describe what settings do or what to do with a setting.
    - Use clear nouns for
        - menus, form field names 

**Cases**

    The **informational** tone is the **most used** one. It is everywhere in the UI, and sometimes seems so normal that we forget about it. Yet, it is very important! 
    
    We use it for naming and describing things:

    - Menus, titles, setting names, hint text, placeholders, introductions, and descriptions

>     - In-product assistance (tooltips, help popovers, guided setups)

Author: ```suggestion
    - In-product assistance (tooltips, help popovers, setup guides, syntax references, etc.)
```
Goes with the suggestion a few lines further

    - Information callouts
    - Confirmation modals
    - Calls to action (buttons and links)
    - Event logs

>     **Documentation**
>     - Reference content
>     - Concepts
>     - How-tos
>     - Troubleshooting

Author: ```suggestion
```
We could drop this part and instead add some of those to the "in-product help" line.

> ## Empathetic

Author: ```suggestion
## Supportive
```
I would call this supportive because I think "empathetic" is almost like a voice trait.

**Attributes**

>     Something might be wrong or definitely wrong and requires attention. Let us help you. Be:

Author: ```suggestion
    Something might be wrong and requires attention. Let us help you. Be:
```

    - Reassuring
    - Serious
    - Urgent

    When using this tone, you may:
    - Be more action-oriented than usual to offer direct paths to users.
    - Focus on what went wrong. Don't be apologetic.
    - Help users learn how something happened, how to prevent it, how to fix it.
    - Provide clear actions, or clearly state it if there's no action needed.

**Cases**

>     The **empathetic** tone is used **sparingly** overall and is usually shown in places that draw the user's attention, for example, through callouts or notifications.

Author: ```suggestion
    The **supportive** tone is used **sparingly** overall and is usually shown in places that draw the user's attention, for example in warnings and error messages.
```

    
    We use it when notifying users of things that require attention, especially in:

    - Errors
    - Licensing or billing-related content
    - Vulnerability and remediation content
    - Warnings and cautions
    - Upgrade and deprecation notices

## Stern

**Attributes**

    We don't advise this action, but you're the master of your destiny. Be:

    - Deterring
    - Heavy with caution
    - Advisory

    When using this tone, you may:
    - Use stronger words that highlight possible negative effects.
    - Format the content in a way that make the risks stand out.

**Cases**

    The **stern** tone is **rarely** used. Reserve it for very specific cases, and for advanced or administrative features.
    
    We use it when insisting on the high risks an action could pose, especially in:

>     - Danger type of warnings

Author: ```suggestion
    - Danger type of warnings
    - Unsupported actions
```

---
id: content_tone
slug: content-tone
sidebar_position: 20
---

# Tone

Adapt the way you write to each situation users can face. 

>  Our products and solutions are very technical, data-oriented, and they're used globally by people from different cultures, abilities, and personalities. 
>  These considerations, amongst others, limit the spectrum of tones we use to some extent, but not entirely. 

Reviewer 2: ```suggestion
```
